tests/test_spark.py

```diff
@@ -175,6 +175,16 @@ def test_jars_flag(service_name=utils.SPARK_SERVICE_NAME):
         args=["--jars {}".format(uploadedJarUrl),
               "--class MultiConfs"])
 
+@pytest.mark.sanity
+def test_packages_flag(service_name=utils.SPARK_SERVICE_NAME):
```

Comment:
`service_name` already defaults to `SPARK_SERVICE_NAME` in `utils.run_tests` no need to provide it here

Reply:
Removed it.